src/Cookies.php

```diff
@@ -316,7 +320,9 @@ protected function _matchPath($domains, $path)
         $ret = [];
 
         foreach ($domains as $dom => $pathsArray) {
-            foreach (array_keys($pathsArray) as $cpath) {
+            /** @var string[] $keys */
+            $keys = array_keys($pathsArray);
```

Comment:
Should be fixable if `$domains` is mapped as `array<string, array<string, mixed>>` (in the parameters)

Reply:
I can't use that type definition, probably is better to check the ´$cpath` value and throw an exception if it isn't a `string`